freqtrade/constants.py

```diff
@@ -176,7 +176,7 @@
                 'order_book_max': {'type': 'integer', 'minimum': 1, 'maximum': 50},
                 'use_sell_signal': {'type': 'boolean'},
                 'sell_profit_only': {'type': 'boolean'},
-                'sell_profit_offset': {'type': 'number', 'minimum': 0.0},
+                'sell_profit_offset': {'type': 'number', 'minimum': -100},
```

Comment:
you can simply remove the whole `'minimum':xxx` key - using -100 is similar to using nothing - and i guess there's no harm in allowing whatever the user selects to provide (even if it's -200)...